lib/Doctrine/Common/Cache/ClearableCache.php

```diff
@@ -29,7 +32,7 @@
 interface ClearableCache
 {
     /**
-     * Deletes all cache entries.
+     * Deletes all cache entries in the current cache namespace.
```

Comment:
Hmm, so if a cache type/adapter doesn't support namespaces, this is pretty much just the `Flushable` cache?